engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -95,6 +97,19 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
+    @Override
+    public void setUri(String nodeUri) {
```

Comment:
The current thinking is that modules instantiate the nodes and then adds them to the graph via addNode. How can the graph be responsible for setting the uri (especially its namespace) if the node is already instantiated and the uri is set on construction?